custom-namespaces.yaml

```diff
@@ -423,26 +390,11 @@ firefox_accounts:
         - channel: release
           table: mozdata.firefox_accounts.fxa_log_device_command_events
   explores:
-    all_event_counts:
-      type: events_explore
-      views:
-        base_view: all_events
-        extended_view: all_events_table
     event_counts:
       type: events_explore
       views:
         base_view: events
-        extended_view: fxa_events_table
-    auth_event_counts:
-      type: events_explore
-      views:
-        base_view: auth_events
-        extended_view: fxa_auth_events_table
```

Comment:
The data catalog doesn't currently show the `firefox_accounts.auth_event_counts` explore either, so I'm not sure how you'd tell what removing it might break.